core/src/mindustry/world/blocks/power/Battery.java

```diff
@@ -1,10 +1,35 @@
 package mindustry.world.blocks.power;
 
+import arc.*;
+import arc.graphics.*;
+import arc.graphics.g2d.*;
+import mindustry.world.*;
+
 public class Battery extends PowerDistributor{
+    public TextureRegion lightsRegion;
+    public Color emptyLightColor = Color.valueOf("6e7080");
+    public Color fullLightColor = Color.valueOf("fb9567");
 
     public Battery(String name){
         super(name);
         outputsPower = true;
         consumesPower = true;
     }
+    
+    @Override
+    public void load(){
+        super.load();
+        lightsRegion = Core.atlas.find(name + "-lights");
```

Comment:
does this texture exist?

Reply:
no, haven't added sprites yet

~~try looking at the past 5 pulls~~

Reply:
i am still yet to pack the sprites